lib/cli.js

```diff
@@ -11,6 +11,7 @@ const utils = require('./utils')
 // VCS implementations
 const BitbucketServer = require('./vcs/bitbucket-server')
 const GitHub = require('./vcs/github')
+const GitHubEnterprise = require('./vcs/github-enterprise')
```

Comment:
I guess with `require` that's fine, but I prefer to treat `require` like `import` and put them all at the top. In this case, I think those modules are small enough that there's not a big need to lazy-load it. If we end up with lots more providers, then yeah, maybe we want something like what you suggested.